.github/ISSUE_TEMPLATE.md

```diff
@@ -3,7 +3,22 @@
 #### Description of the problem
 
 <!-- If you can, it's helpful to provide self-contained example of some code
-that reproduces the bug. If not, a link to a larger example is also helpful. -->
+that reproduces the bug (you can uncomment the @coqbot minimize block below).
+If not, a link to a larger example is also helpful. -->
+
+<!-- @coqbot minimize
```

Comment:
```suggestion
<!-- @coqbot: minimize
```

As just discovered, it won't work if there's no `:`

Reply:
Oops, thanks!